[2024-10-16T09:40:02.313+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: exchange_rate_usd_rus_dag.insert_rate scheduled__2024-08-31T00:00:00+00:00 [queued]>
[2024-10-16T09:40:02.338+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: exchange_rate_usd_rus_dag.insert_rate scheduled__2024-08-31T00:00:00+00:00 [queued]>
[2024-10-16T09:40:02.340+0000] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2024-10-16T09:40:02.348+0000] {taskinstance.py:1284} INFO - Starting attempt 1 of 1
[2024-10-16T09:40:02.355+0000] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2024-10-16T09:40:02.423+0000] {taskinstance.py:1304} INFO - Executing <Task(PostgresOperator): insert_rate> on 2024-08-31 00:00:00+00:00
[2024-10-16T09:40:02.439+0000] {standard_task_runner.py:55} INFO - Started process 4202 to run task
[2024-10-16T09:40:02.457+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'exchange_rate_usd_rus_dag', 'insert_rate', 'scheduled__2024-08-31T00:00:00+00:00', '--job-id', '355', '--raw', '--subdir', 'DAGS_FOLDER/RUS_DAG.py', '--cfg-path', '/tmp/tmp9c8jwp4v']
[2024-10-16T09:40:02.459+0000] {standard_task_runner.py:83} INFO - Job 355: Subtask insert_rate
[2024-10-16T09:40:02.629+0000] {task_command.py:389} INFO - Running <TaskInstance: exchange_rate_usd_rus_dag.insert_rate scheduled__2024-08-31T00:00:00+00:00 [running]> on host 63ee4fa64e74
[2024-10-16T09:40:03.777+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:205: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2024-10-16T09:40:05.555+0000] {taskinstance.py:1513} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=exchange_rate_usd_rus_dag
AIRFLOW_CTX_TASK_ID=insert_rate
AIRFLOW_CTX_EXECUTION_DATE=2024-08-31T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-08-31T00:00:00+00:00
[2024-10-16T09:40:05.566+0000] {sql.py:253} INFO - Executing: INSERT INTO
    currency_exchange_rates
VALUES ('USD', 'RUS', None, '2024-08-31')
ON CONFLICT (base, currency, date) DO
    UPDATE
        SET rate = excluded.rate;
[2024-10-16T09:40:05.597+0000] {base.py:73} INFO - Using connection ID 'postgres_default' for task execution.
[2024-10-16T09:40:06.321+0000] {base.py:73} INFO - Using connection ID 'postgres_default' for task execution.
[2024-10-16T09:40:06.369+0000] {sql.py:364} INFO - Running statement: INSERT INTO
    currency_exchange_rates
VALUES ('USD', 'RUS', None, '2024-08-31')
ON CONFLICT (base, currency, date) DO
    UPDATE
        SET rate = excluded.rate;, parameters: None
[2024-10-16T09:40:06.424+0000] {taskinstance.py:1772} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/common/sql/operators/sql.py", line 261, in execute
    return_last=self.return_last,
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/common/sql/hooks/sql.py", line 338, in run
    self._run_command(cur, sql_statement, parameters)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/common/sql/hooks/sql.py", line 369, in _run_command
    cur.execute(sql_statement)
psycopg2.errors.UndefinedColumn: column "none" does not exist
LINE 3: VALUES ('USD', 'RUS', None, '2024-08-31')
                              ^

[2024-10-16T09:40:06.528+0000] {taskinstance.py:1327} INFO - Marking task as FAILED. dag_id=exchange_rate_usd_rus_dag, task_id=insert_rate, execution_date=20240831T000000, start_date=20241016T094002, end_date=20241016T094006
[2024-10-16T09:40:06.692+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 355 for task insert_rate (column "none" does not exist
LINE 3: VALUES ('USD', 'RUS', None, '2024-08-31')
                              ^
; 4202)
[2024-10-16T09:40:06.834+0000] {local_task_job.py:159} INFO - Task exited with return code 1
[2024-10-16T09:40:07.003+0000] {taskinstance.py:2582} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-10-16T09:49:45.108+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: exchange_rate_usd_rus_dag.insert_rate scheduled__2024-08-31T00:00:00+00:00 [queued]>
[2024-10-16T09:49:45.123+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: exchange_rate_usd_rus_dag.insert_rate scheduled__2024-08-31T00:00:00+00:00 [queued]>
[2024-10-16T09:49:45.124+0000] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2024-10-16T09:49:45.124+0000] {taskinstance.py:1284} INFO - Starting attempt 1 of 1
[2024-10-16T09:49:45.125+0000] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2024-10-16T09:49:45.146+0000] {taskinstance.py:1304} INFO - Executing <Task(PostgresOperator): insert_rate> on 2024-08-31 00:00:00+00:00
[2024-10-16T09:49:45.152+0000] {standard_task_runner.py:55} INFO - Started process 6997 to run task
[2024-10-16T09:49:45.157+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'exchange_rate_usd_rus_dag', 'insert_rate', 'scheduled__2024-08-31T00:00:00+00:00', '--job-id', '1212', '--raw', '--subdir', 'DAGS_FOLDER/RUS_DAG.py', '--cfg-path', '/tmp/tmpxejqcvz1']
[2024-10-16T09:49:45.161+0000] {standard_task_runner.py:83} INFO - Job 1212: Subtask insert_rate
[2024-10-16T09:49:45.262+0000] {task_command.py:389} INFO - Running <TaskInstance: exchange_rate_usd_rus_dag.insert_rate scheduled__2024-08-31T00:00:00+00:00 [running]> on host 63ee4fa64e74
[2024-10-16T09:49:45.326+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:205: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2024-10-16T09:49:45.426+0000] {taskinstance.py:1513} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=exchange_rate_usd_rus_dag
AIRFLOW_CTX_TASK_ID=insert_rate
AIRFLOW_CTX_EXECUTION_DATE=2024-08-31T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-08-31T00:00:00+00:00
[2024-10-16T09:49:45.428+0000] {sql.py:253} INFO - Executing: INSERT INTO
    currency_exchange_rates
VALUES ('USD', 'RUB', 90.68668009, '2024-08-31')
ON CONFLICT (base, currency, date) DO
    UPDATE
        SET rate = excluded.rate;
[2024-10-16T09:49:45.444+0000] {base.py:73} INFO - Using connection ID 'postgres_default' for task execution.
[2024-10-16T09:49:45.745+0000] {base.py:73} INFO - Using connection ID 'postgres_default' for task execution.
[2024-10-16T09:49:45.754+0000] {sql.py:364} INFO - Running statement: INSERT INTO
    currency_exchange_rates
VALUES ('USD', 'RUB', 90.68668009, '2024-08-31')
ON CONFLICT (base, currency, date) DO
    UPDATE
        SET rate = excluded.rate;, parameters: None
[2024-10-16T09:49:45.756+0000] {sql.py:373} INFO - Rows affected: 1
[2024-10-16T09:49:45.770+0000] {taskinstance.py:1327} INFO - Marking task as SUCCESS. dag_id=exchange_rate_usd_rus_dag, task_id=insert_rate, execution_date=20240831T000000, start_date=20241016T094945, end_date=20241016T094945
[2024-10-16T09:49:45.827+0000] {local_task_job.py:159} INFO - Task exited with return code 0
[2024-10-16T09:49:45.849+0000] {taskinstance.py:2582} INFO - 0 downstream tasks scheduled from follow-on schedule check
